code/modules/clothing/neck/_neck.dm

```diff
@@ -72,6 +72,9 @@
 					if(!(M.failed_last_breath || M.losebreath))
 						lung_strength = "healthy"
 
+			if(M.stat == DEAD && heart && world.time - M.timeofdeath < DEFIB_TIME_LIMIT * 10)
+				heart_strength = "<span class='boldannounce'>a faint, fluttery</span>"
```

Comment:
"You faintly hear a faint..."

It's faint is it? Never would have guessed.

Reply:
If I said that I thought it wasn't amusing, I'd be *feinting* the question.